godot-macros/src/method_registration/virtual_method_callback.rs

```diff
@@ -0,0 +1,37 @@
+/*
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at https://mozilla.org/MPL/2.0/.
+*/
+
+use proc_macro2::{Ident, TokenStream as TokenStream2};
+use quote::quote;
+use venial::Function;
+
+use crate::method_registration::get_ptrcall;
+
+/// Returns a C function which acts as the callback when a virtual method of this instance is invoked.
+//
+// There are currently no virtual static methods. Additionally, virtual static methods dont really make a lot
+// of sense. Therefore there is no need to support them.
+pub fn gdext_virtual_method_callback(
+    class_name: &Ident,
+    method_signature: &Function,
+) -> TokenStream2 {
+    let ptrcall = get_ptrcall(class_name, method_signature);
+
+    quote! {
+        Some({
+            use godot::sys;
+
+            unsafe extern "C" fn function(
+                instance_ptr: sys::GDExtensionClassInstancePtr,
+                args: *const sys::GDExtensionConstTypePtr,
+                ret: sys::GDExtensionTypePtr,
+            ) {
+                #ptrcall
+            }
+            function
+        })
```

Comment:
Instead of having `Some(..)` around the whole block, could you just change the last line inside the block to `Some(function)`?